backend/protondrive/protondrive.go

```diff
@@ -271,18 +271,34 @@ func getConfigMap(m configmap.Mapper) (uid, accessToken, refreshToken, saltedKey
 	if accessToken, ok = m.Get(clientAccessTokenKey); !ok {
 		return
 	}
+	if len(accessToken) == 0 {
```

Comment:
I think now that we've made config entries for them, you should just use the `Options` parser and lose this function entirely.

Reply:
rclone beta testers are used to having to remake their config files so don't worry about it. From now on we'll have to be a bit more careful as I try not to break people's configs unless I absolutely have to and even then rclone should ERROR with instructions.

PS Tag any PRs you want in the point release with `point release candidate`!